index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./assets/css/styles.css">
+    <title>First Project</title>
+</head>
+<body>
+    <header>
+        <div class="topHeader">
+            <div class="logo">
+                <a href="#" rel="noopener" target="_blank">
+                   My Logo
+                </a>
+            </div>    
+            <nav> 
+                <a href="#" rel="noopener" target="_blank">
+                    <img src="/assets/images/breadcrumb.png" alt="">
+                </a>    
+            </nav>
+        </div>
+    </header>    
+    <section class="content">
+        <h1>I’m Alexandra. <br />Glad to see you!</h1>
+        <div class="paragraph">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, dont hesitate to contact me.</div>
+        <a href="#">LET’S CONNECT</a>
+        <ul class="social-media">
+            <li><a href="https://twitter.com/javiersistemas" rel="noopener" target="_blank"><img src="/assets/images/twitter.svg" alt=""></a></li>
+            <li><a href="https://www.linkedin.com/in/javier-jesus-martinez-fari%C3%B1as-b71b3968/" rel="noopener" target="_blank"><img src="/assets/images/linkedin.svg" alt=""></a></li>
+            <li><a href="#" target="_blank"><img src="/assets/images/medium.svg" alt=""></a></li>
+            <li><a href="https://github.com/jlcarrascof" rel="noopener" target="_blank"><img src="/assets/images/github.svg" alt=""></a></li>
+            <li><a href="#" target="_blank"><img src="/assets/images/angel-list.svg" alt=""></a></li>
```

Comment:
you can fix this issue by adding the dot to refer to current directory  src="./assets/images/linkedin.svg"

Reply:
Hi, dear @souad988 I'm doing the changes. Thanks for your explicit feedback, dear friend.

Javier.-